server_src/lib/serverCommands.js

```diff
@@ -131,7 +131,12 @@ class SystemCommands {
           "<b class='text-danger'>Cancel:</b> This option will cancel the update process and not install the required dependencies. No update will run and manual intervention by the user is required. <br><br>";
         return clientResponse;
       } else {
-        await installMissingNpmDependencies();
+        if (missingPackagesList.length > 0) {
+          for (let i = 0; i > missingPackagesList.length; i++) {
+            await installNpmDependencies(missingPackagesList[i]);
+          }
+        }
+
```

Comment:
Just changed here to cycle through the dependency and fire any missing packages to the updated function